db/migrate/20200401172023_create_published_pages.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreatePublishedPages < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
given it doesn't soon like we use the id of this model maybe we could get rid of the auto increment and store the topic_id  in the id column as primary_key

Reply:
I thought of this but I considered in the future we might be publishing other things and I didn't want to "lock in" to a topic just yet.

For now the extra data for the key is minimal and I think is OK.